setuptools/tests/test_setuptools.py

```diff
@@ -64,6 +65,7 @@ def f1():
         # recognized name, not assigned
         dep.extract_constant(fc, 'z', -1) is None
 
+    @py2_only
```

Comment:
`dep.find_module` appears to be a tested part of the public API, we cannot make this test `py2_only` unless we intend to *remove* it for Python 3 users.

If we're planning to remove it for Python 3 users, we should probably remove it for Python 2 users as well, and we should go through a deprecation cycle.

Reply:
If you want to keep it, i can make it working with importlib on py3+, i thought the `find_spec` solves the problem so this is not needed on py3+